Converters/ArcGIS/Speckle.Converters.ArcGIS3/ToSpeckle/Raw/PointToSpeckleConverter.cs

```diff
@@ -15,14 +14,14 @@ public PointToSpeckleConverter(IConverterSettingsStore<ArcGISConversionSettings>
     _settingsStore = settingsStore;
   }
 
-  public SOG.Point Convert(MapPoint target)
+  public SOG.Point Convert(ACG.MapPoint target)
   {
     try
     {
       // reproject to Active CRS
       if (
-        GeometryEngine.Instance.Project(target, _settingsStore.Current.ActiveCRSoffsetRotation.SpatialReference)
-        is not MapPoint reprojectedPt
+        ACG.GeometryEngine.Instance.Project(target, _settingsStore.Current.ActiveCRSoffsetRotation.SpatialReference)
+        is not ACG.MapPoint reprojectedPt
       )
       {
         throw new ValidationException(
```

Comment:
i know this was pre-existing but curious what is the `ValidationException`, don't we suppose to use some conversion typed exceptions